II.pass-tests/04-find-needle/find-needle.test.js

```diff
@@ -4,6 +4,14 @@ test("Find the needle", function() {
   var words = ["house", "train", "slide", "needle", "book"];
   var expected = 3;
 
-  var output = findTheNeedle(words);
+  var output = findTheNeedle(words, "needle");
+  expect(output).toEqual(expected);
+});
+
+test("Find the plant", function() {
```

Comment:
I agree with the idea of including more tests. The current exercise is ambiguous and `words.indexOf("needle")` is a valid implementation

Reply:
Yeah I agree. I was going back and forth on whether to rename the exercise entirely (`find-word`?). What do you think?

Reply:
sure, we can fix it in upcoming PRs but this one looks gtm